inspirehep/dojson/geo_helpers.py

```diff
@@ -0,0 +1,420 @@
+    'SAINT LUCIA': 'LC',
+    'SAINT MARTIN (FRENCH PART)': 'MF',
+    'SAINT PIERRE AND MIQUELON': 'PM',
+    'SAINT VINCENT AND THE GRENADINES': 'VC',
+    'SAMOA': 'WS',
+    'SAN MARINO': 'SM',
+    'SAO TOME AND PRINCIPE': 'ST',
+    'SAUDI ARABIA': 'SA',
+    'SENEGAL': 'SN',
+    'SERBIA': 'RS',
+    'SERBIA AND MONTENEGRO': 'AB',
+    'SEYCHELLES': 'SC',
+    'SIERRA LEONE': 'SL',
+    'SINGAPORE': 'SG',
+    'SINT MAARTEN (DUTCH PART)': 'SX',
+    'SLOVAKIA': 'SK',
+    'SLOVENIA': 'SI',
+    'SOLOMON ISLANDS': 'SB',
+    'SOMALIA': 'SO',
+    'SOUTH AFRICA': 'ZA',
+    'SOUTH GEORGIA AND THE SOUTH SANDWICH ISLANDS': 'GS',
+    'SOUTH SUDAN': 'SS',
+    'SPAIN': 'ES',
+    'SRI LANKA': 'LK',
+    'SUDAN': 'SD',
+    'SURINAME': 'SR',
+    'SVALBARD AND JAN MAYEN': 'SJ',
+    'SWAZILAND': 'SZ',
+    'SWEDEN': 'SE',
+    'SWITZERLAND': 'CH',
+    'SYRIAN ARAB REPUBLIC': 'SY',
+    'TAIWAN, PROVINCE OF CHINA': 'TW',
+    'TAJIKISTAN': 'TJ',
+    'TANZANIA, UNITED REPUBLIC OF': 'TZ',
+    'THAILAND': 'TH',
+    'TIMOR-LESTE': 'TL',
+    'TOGO': 'TG',
+    'TOKELAU': 'TK',
+    'TONGA': 'TO',
+    'TRINIDAD AND TOBAGO': 'TT',
+    'TUNISIA': 'TN',
+    'TURKEY': 'TR',
+    'TURKMENISTAN': 'TM',
+    'TURKS AND CAICOS ISLANDS': 'TC',
+    'TUVALU': 'TV',
+    'UGANDA': 'UG',
+    'UKRAINE': 'UA',
+    'UNITED ARAB EMIRATES': 'AE',
+    'UNITED KINGDOM': 'GB',
+    'UNITED STATES': 'US',
+    'UNITED STATES MINOR OUTLYING ISLANDS': 'UM',
+    'URUGUAY': 'UY',
+    'USSR': 'SU',
+    'UZBEKISTAN': 'UZ',
+    'VANUATU': 'VU',
+    'VENEZUELA, BOLIVARIAN REPUBLIC OF': 'VE',
+    'VIET NAM': 'VN',
+    'VIRGIN ISLANDS, BRITISH': 'VG',
+    'VIRGIN ISLANDS, U.S.': 'VI',
+    'WALLIS AND FUTUNA': 'WF',
+    'WESTERN SAHARA': 'EH',
+    'YEMEN': 'YE',
+    'YUGOSLAVIA': 'YU',
+    'ZAMBIA': 'ZM',
+    'ZIMBABWE': 'ZW',
+}
+
+iso_code_to_country_name = {v: k for k, v in country_to_iso_code.items()}
+
+countries_alternative_codes = {
+    'CN': ['CHINA'],
+    'FI': ['FN'],
+    'FR': ['FX'],
+    'GB': ['UK'],
+    'TL': ['TP'],
+    'CD': ['ZR']
+}
+
+countries_alternative_spellings = {
+    'AU': ['SAUSTRALIA'],
+    'AM': ['REPUBLIC OF ARMENIA'],
+    'BA': ['BOSNIA-HERZEGOVINA', 'BOSNIA'],
+    'BH': ['KINGDOM OF BAHRAIN'],
+    'BJ': ['REPUBLIC OF BENIN'],
+    'BO': ['BOLIVIA'],
+    'BR': ['BRASIL', 'SC BRAZIL'],
+    'CA': ['BC CANADA', 'BRITISH COLUMBIA'],
+    'CH': ['SWITZ', 'SWITZLERAND', 'SWITZERLAND (CERN)', 'SWITERLAND'],
+    'CN': ['PR CHINA'],
+    'CS': ['CZECHSOLVAKIA'],
+    'CZ': ['PRAGUE'],
+    'DE': ['DEUTSCHLAND', 'WEST GERMANY', 'EAST GERMANY', 'BAVARIA',
+           'GERMANY (DESY)'],
+    'ES': ['CANARY ISLANDS', 'MADRID'],
+    'FR': ['CORSICA'],
+    'GR': ['CRETE'],
+    'GB': ['UK', 'ENGLAND', 'ENG', 'SCOTLAND', 'WALES', 'SCOTLAND/UK',
+           'NORTHERN IRELAND', 'LONDON'],
+    'ID': ['BALI'],
+    'IL': ['JERUSALEM'],
+    'IR': ['IRAN'],
+    'IT': ['ITALIE', 'ITALIA', 'NAPOLI', 'MESSINA', 'SARDINIA', 'SICILY'],
```

Comment:
Ok, I don't mind the idea of a dummy geocoder which works in the 90% of cases. I don't agree with putting those methods in `utils`, since I don't think they have other users... maybe `inspirehep_dojson_utils` is a better place?